06_predict_survival/nbconverted/plot_survival_results.py

```diff
@@ -196,12 +209,335 @@
 sns.set({'figure.figsize': (28, 6)})
 sns.set_context('notebook', font_scale=1.5)
 
-ax = plu.plot_heatmap(heatmap_df,
-                      all_results_df.reset_index(drop=True),
+ax = plu.plot_heatmap(me_heatmap_df,
+                      me_all_results_df.reset_index(drop=True),
+                      different_from_best=True,
+                      raw_results_df=raw_results_df,
+                      metric='cindex',
+                      id_name='cancer_type',
+                      scale=(-0.1, 0.4))
+
```

Comment:
Probably not a big deal, but if, for example, the me_27k data for BLCA wasn't supposed to be in a box, it would still look like it is supposed to be since its four neighbors are 

![image](https://user-images.githubusercontent.com/15317110/132385649-90d03400-76d2-4a5a-82ad-935b4db167b8.png)

Reply:
Yeah, you're totally right - this is something I noticed a while ago and haven't gotten around to fixing. I agree it would be better if I could move the boxes in a bit (so they would be visually more distinct for each heatmap square) but it's a bit finicky to pull it off in matplotlib.

Created issue #63 for this, hopefully I can get to it in the next week or two.